components/actionbutton/metadata/actionbutton.yml

```
@@ -15,9 +15,26 @@ sections:
       ### Version 6.0.0
       #### Spectrum 2 release
       Action button now uses Spectrum 2 tokens and specifications. A few notable changes:
-      - The mod custom property `--mod-line-height-100` has been renamed to `--mod-button-line-height`.
-      - The mod custom property `--mod-sans-font-family-stack` has been renamed to `--mod-button-font-family`.
-      - The mod custom property `--mod-animation-duration-100` has been renamed to `--mod-button-animation-duration`.
+      - Medium is now the default size and `.spectrum-Switch--sizeM` has been removed.
+      - Includes the Spectrum 2 down state transform.
+      - The component border was not removed to continue support for Windows High Contrast Mode (WHCM), but the color was adjusted to `transparent` and the mod custom properties were removed so as not to interfere with WHCM accessibility.
```

```suggestion
      - The component border was not removed and its color was changed to `transparent` in order to continue support for Windows High Contrast / forced colors, which still shows a border.
```
An idea for rewording this to clarify.

---

```suggestion
      - The component border was not removed and its color was changed to `transparent` in order to continue support for Windows High Contrast / forced colors, which still shows a border. The mod custom properties for border were removed to avoid interference with Windows High Contrast / forced colors accessibility.
```